snippets/about-node-selectors.adoc

```diff
@@ -0,0 +1,10 @@
+// Snippets included in the following assemblies and modules:
+//
+// * nodes/scheduling/nodes-scheduler-node-selectors.adoc
+// * logging/scheduling_resources/logging-node-selection.adoc
+
+:_mod-docs-content-type: SNIPPET
+
+A _node selector_ specifies a map of key/value pairs that are defined using custom labels on nodes and selectors specified in pods.
+
+For the pod to be eligible to run on a node, the pod must have the same key/value node selector as the label on the node.
```

Comment:
... the pod must match keys and values of  labels on the node.....

something to this affect.  They don't have to match all the node labels, just a subset of them

Reply:
This is preexisting content that is used elsewhere in the docs - I have just moved it into a snippet so that we can reuse it in our module. Per the other comments, I'd prefer to follow up on any additions to the conceptual content another time, and keep this PR to required fixes in the config and restructuring.